app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayViewModel.kt

```diff
@@ -65,4 +66,13 @@ class NavigationOverlayViewModel(
                     else -> R.id.turboButton
                 }
             }
+
+    val toolBarFocusListener = View.OnKeyListener { v: View?, keyCode: Int, _: KeyEvent? ->
+        if (keyCode == KeyEvent.KEYCODE_DPAD_LEFT) {
+            v?.requestFocus()
+            true
+        } else {
+            false
```

Comment:
I think we should probably take a different approach to this, for a few reasons.  There's a rule of thumb in Android dev that `requestFocus` should only be used after everything else has failed, for a number of reasons (it's uninutitive to change user focus, depending on the exact timing it might show a flash of focus elsewhere before moving, etc).  I also don't love that we're handling the same kind of problem differently for different views (some are using `nextFocus*Id`, some are intercepting dpad events).  I think we need to find a different solution to this.